label/README.rst

```diff
@@ -0,0 +1,75 @@
+====================
```

Comment:
Please update the readme file using the oca script (that will merge the files present in the readme folder)